packages/ast-spec/tests/util/serialize-error.ts

```diff
@@ -13,16 +13,12 @@ export function serializeError(error: unknown, contents: string): unknown {
     location: { start, end },
   } = error;
 
-  return (
-    name +
-    '\n' +
-    codeFrameColumns(
-      contents,
-      {
-        start: { line: start.line, column: start.column + 1 },
-        end: { line: end.line, column: end.column + 1 },
-      },
-      { highlightCode: false, message },
-    )
-  );
+  return `${name}\n${codeFrameColumns(
```

Comment:
This is one of those cases that I think benefits from having the interpolated expression be assigned a variable

Reply:
Here we don't need to—if you change that to a line break, the second line would fit nicely on one line and it would be equally readable.